Gruntfile.js

```diff
@@ -1,63 +1,187 @@
-'use strict';
+var request = require('superagent');
 
 module.exports = function(grunt) {
+  'use strict';
+
+  require('load-grunt-tasks')(grunt);
+
   grunt.initConfig({
-    'git-describe': {
-      options: {
+    pkg: grunt.file.readJSON('package.json'),
```

Comment:
Is this required for the `bump` task? I'm not seeing any new configuration in `package.json`, so wanted to check if this is necessary.